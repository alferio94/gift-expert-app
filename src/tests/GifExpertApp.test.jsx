import { render, screen, fireEvent } from '@testing-library/react'
import { GifExpertApp } from '../GifExpertApp';

describe('GiftExpertApp', () =>
{
    test('renders correctly', () =>
    {
        const { container } = render(<GifExpertApp />);
        expect(container).toMatchSnapshot();
    });
})