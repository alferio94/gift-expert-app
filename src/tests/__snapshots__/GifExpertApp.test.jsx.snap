// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GiftExpertApp renders correctly 1`] = `
<div>
  <h1>
    Gif Expert App
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Buscar"
      type="text"
      value=""
    />
  </form>
  <h3 />
  <p>
    Loading...
  </p>
</div>
`;
